Assets/Scripts/Managers/HighlightManager.cs

```diff
@@ -131,6 +148,29 @@ private void ResetSelection(GameState gameState)
                 }
 
             }
+            // temp implementation
+            if (selectedPlayer == null)
+            {
+                return;
+            } else
+            {
+                selectedPlayer.UnRenderHand();
+            }
+        } 
+    }
+
+    // This method is actually redundant since you would HAVE to see the updated deck if you have a player selected.
+    public static bool RenderHandIfAppropriate(PlayerClass player)
+    {
+        if (currentPlayerDisplayDeck == player)
+        {
+            player.PleaseRenderMyHand();
+            return true;
+        } else
+        {
+            return false;
         }
     }
```

Comment:
If the comment is correct and the method is actually redundant, consider refactoring/removing it?

Reply:
You can save lines by
`cond = curr... == player
if (cond) {
render()
}
return cond;`